src/main/java/jackson/utils/Ui.java

```diff
@@ -0,0 +1,213 @@
+            break;
+        case "event":
+            output += """
+            event [task-name] /from [start-date] /to [end-date]
+            All dates must be in DD-MM-YYYY HH:MM format (HH:MM is optional)""";
+            break;
+        case "list":
+            output += "list";
+            break;
+        case "mark":
+            output += "mark [index]";
+            break;
+        case "unmark":
+            output += "unmark [index]";
+            break;
+        case "delete":
+            output += "delete [index]";
+            break;
+        case "find":
+            output += "find [keywords]";
+            break;
+        case "bye":
+            output += "bye";
+            break;
+        default:
+            output += "Unknown error...";
+            break;
+        }
+        return output + "\n";
+    }
+
+    /**
+     * Returns valid indices when invalid index is entered.
+     * @param taskList {@code OutOfListException} exception encountered.
+     * @return String response.
+     */
+    public String printIndexGuide(TaskList taskList) {
+        int size = taskList.getSize();
+        String output = String.format("Alamak, you got %d items on the list only leh...\n", size);
+
+        // different responses based on taskList size
+        if (size == 0) {
+            output += "You've got no items in the list! Add some stuff first!";
+        } else if (size == 1) {
+            output += "Enter 1 to mark/unmark/delete the task in the list!";
+        } else {
+            output += String.format("Enter a number between 1 and %d when marking/unmarking/deleting tasks!\n", size);
+        }
+        return output + "\n";
+    }
+
+    /**
+     * Returns generic error message when other Exceptions occur.
+     * @param e {@code Exception} encountered during runtime.
+     * @return String response.
+     */
+    public String printUnknownError(Exception e) {
+        return String.format("Oops! Something went wrong!\n%s\n", e.toString());
+    }
+
+    /**
+     * Returns chatbot response when adding task to list.
+     * @param task {@code Task} object to be added to taskList.
+     * @param taskList {@code TaskList} object task is to be added to.
+     * @return String response.
+     */
+    public String printAddList(Task task, TaskList taskList) {
+        String output = "";
+        output += "Ya la, adding this task to your list!\n";
+        output += String.format("\t%s\n", task);
+        output += String.format("You now got %s tasks in your list leh\n", taskList.getSize());
+        return output;
+    }
+
+    /**
+     * Returns chatbot response when deleting task from list.
+     * @param task {@code Task} object to be deleted from taskList.
+     * @param taskList {@code TaskList} object task is to be deleted from.
+     * @return String response.
+     */
+    public String printDeleteList(Task task, TaskList taskList) {
```

Comment:
Maybe we could rename this to show that we're printing after deleting something on the list? Right now when I first read it, I thought it is print, then delete list.

Reply:
fair point! what i had in mind was that the command given was "deleteList", thus to print that command the method should be called "printDeleteList", but i get your point. will change it and thanks for the review :+1: